cli/src/lib.rs

```diff
@@ -926,7 +926,12 @@ fn init(
     if force {
         fs::create_dir_all(&project_name)?;
     } else {
-        fs::create_dir(&project_name)?;
+        let path = std::env::current_dir()?;
+        let new_path = path.join(&rust_name);
+
+        if let Err(_) = fs::read_dir(new_path) {
+            fs::create_dir(&project_name)?;
+        };
```

Comment:
This change is redundant because [`std::fs::create_dir`](https://doc.rust-lang.org/std/fs/fn.create_dir.html) already fails if the directory exists (see [Errors section](https://doc.rust-lang.org/std/fs/fn.create_dir.html#errors)).